test/validators.js

```diff
@@ -11621,6 +11621,21 @@ describe('Validators', () => {
         '1234577A',
         '1234577JA'],
     });
+    test({
+      validator: 'isTaxID',
+      args: ['en-IN'],
+      valid: [
+        'AAAAA1111A',
+        'BBBBB1111B',
+        'CCCCC1111C',
+        'FFFFF0001F',
+        'ZZZFZ9999Z'],
+      invalid: [
+        'DDDDD1111D',
+        '1234567890',
+        'ABCDEFGHIJ',
+        'ABCDE1234F'],
```

Comment:
I think this test case should also be included to make sure that the numbers start at 0001.

```suggestion
        'ABCDE1234F',
        'AAAAA0000A'],
```

Reply:
Modified regex to start numbers from 0001 and added test case for the same.

Reply:
Glad we caught that :)